packages/editor/src/components/rich-text/list-edit.js

```diff
@@ -70,52 +70,54 @@ export const ListEdit = ( { editor, onTagNameChange, tagName, onSyncDOM } ) => (
 			} }
 		/>
 		<BlockFormatControls>
-			<Toolbar
-				controls={ [
-					{
-						icon: 'editor-ul',
-						title: __( 'Convert to unordered list' ),
-						isActive: isActiveListType( editor, 'ul', tagName ),
-						onClick() {
-							if ( isListRootSelected( editor ) ) {
-								onTagNameChange( 'ul' );
-							} else {
-								editor.execCommand( 'InsertUnorderedList' );
-								onSyncDOM();
-							}
+			<div className="list-format-toolbar">
+				<Toolbar
```

Comment:
While not documented, the Toolbar component accepts a `className` prop which will be applied in addition to its own default className.

https://github.com/WordPress/gutenberg/blob/652a662a8aba2b621978871e56706990d5deb653/packages/components/src/toolbar/index.js#L40